### PR TITLE
Correcting the definition for crosscomponents

### PR DESCRIPTION
**Pull request by ramarag**
@rahku  @gkhanna79  PTAL

### CODE REVIEW

**Comment by ramarag** [comment]
@dotnet-bot test Windows_NT arm64 Debug
@dotnet-bot test Windows_NT arm64 Checked
@dotnet-bot test Windows_NT arm64 Release

File: clrdefinitions.cmake [hunk 1]
```diff
@@ -7,7 +7,7 @@ endif()
 if (WIN32)
   set(FEATURE_EVENT_TRACE 1)
 endif()
-if(CLR_CMAKE_PLATFORM_LINUX AND CLR_CMAKE_PLATFORM_ARCH_AMD64)
+if(CLR_CMAKE_PLATFORM_LINUX AND CLR_CMAKE_TARGET_ARCH_AMD64)
   set(FEATURE_EVENT_TRACE 1)
```
Comment (rahku) [inline clrdefinitions.cmake]: should this also get enabled for target_arch_arm64? And why it was not enabled for other arch (arm, x86)

**Comment by rahku** [comment]
Looks good

File: clrdefinitions.cmake (hunk 1 above)
Comment (ramarag) [inline clrdefinitions.cmake]: This is enabling Eventing on Linux, we have only explored this feature for AMD64.
I think Arm64 for linux is still work in progress. And for arm there is no native eventing story yet

**Pull request merged**